orbstation/code/objects/items/food_and_drink/produce_orders.dm

```diff
@@ -0,0 +1,20 @@
+//FRUITS AND VEGGIES
+/datum/orderable_item/veggies/korta_nut
+	name = "Korta Nut"
+	item_path = /obj/item/food/grown/korta_nut
+	cost_per_order = 10
```

Comment:
10 is the default so you dont actually need to list it but that doesnt really matter i think